ServerRecording/README.MD

```diff
@@ -0,0 +1,66 @@
+---
+page_type: sample
+languages:
+- C#
+products:
+- azure
+- azure-communication-services
+---
+
+[![Deploy to Azure](https://aka.ms/deploytoazurebutton)](https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure-Samples%2Fcommunication-services-web-calling-hero%2Fmain%2Fdeploy%2Fazuredeploy.json)
+
+# Recording APIs Sample
+
+This is a sample application to show how the Azure Communication Services server calling SDK can be used to build a call recording feature.
+
+It's a C# based application powered by Dot net core to connect this application with Azure Communication Services.
+
+A separate branch with end to end implementation is [available](https://github.com/Azure-Samples/communication-services-web-calling-hero/tree/public-preview). It's a public preview branch and uses beta SDKs that are not meant for production use. Please use the main branch sample for any production scenarios.
+
+## Prerequisites
+
+- Create an Azure account with an active subscription. For details, see [Create an account for free](https://azure.microsoft.com/free/?WT.mc_id=A261C142F)
+- [Visual Studio (2019 and above)](https://visualstudio.microsoft.com/vs/)
+- [.NET Core 3.1](https://dotnet.microsoft.com/download/dotnet-core/3.1) (Make sure to install the version that corresponds with your visual studio instance, 32 vs 64 bit)
+- Create an Azure Communication Services resource. For details, see [Create an Azure Communication Resource](https://docs.microsoft.com/azure/communication-services/quickstarts/create-communication-resource). You'll need to record your resource **connection string** for this quickstart.
+- Create a webhook and subscribe to the recording events. For details, see [Create webhook](https://docs.microsoft.com/azure/communication-services/quickstarts/voice-video-calling/download-recording-file-sample)
+
+## Code structure
+
+- ./ServerRecording/Controllers : Server app core logic for calling recording APIs using Azure Communication Services server calling SDK
+- ./ServerRecording/Program.cs : Entry point for the server app program logic
+- ./ServerRecording/Startup.cs : Entry point for the server app startup logic
+
+## Before running the sample for the first time
+1. Get the `Connection String` from the Azure portal. For more information on connection strings, see [Create an Azure Communication Resources](https://docs.microsoft.com/azure/communication-services/quickstarts/create-communication-resource)
+2. Once you get the config keys add the keys to the **Calling/appsetting.json**  file found under the Calling folder.
+	- Input your connection string in the variable: `Connectionstring`
```

Comment:
Since there a multiple resources involved with multiple "ConnectionStrings", it may be a good idea to differentiate "Connectionstring" by calling it "AcsConnectionString" instead

Reply:
Updated